streams/src/test/java/org/apache/kafka/streams/state/internals/InMemoryLRUCacheStoreTest.java

```diff
@@ -44,7 +45,7 @@
             Deserializer<K> keyDeser = (Deserializer<K>) context.keyDeserializer();
```

Comment:
I think we forgot to update `ProcessorContext` to use the unified `Serde`, too.

That is, we should have `context.keySerde()` instead of separate `context.keySerializer()` and `context.keyDeserializer()` IMHO.